packages/api/AlvTime.Business/InvoiceRate/InvoiceRateService.cs

```diff
@@ -52,6 +55,78 @@ public async Task<decimal> GetEmployeeInvoiceRateForPeriod(DateTime fromDate, Da
         return billableHours / availableHours;
     }
 
+    public async Task<InvoiceStatisticsDto> GetEmployeeInvoiceStatisticsByMonth(DateTime fromDate, DateTime toDate)
+    {
+        var user = await _userContext.GetCurrentUser();
+        var adjustedFromDate = GetUserStartDateOrFromDate(fromDate, user);
+        adjustedFromDate = adjustedFromDate.AddDays(-adjustedFromDate.Day + 1);
```

Comment:
Dette er kun for å få første dag i måneden, sant? Er det en mer intuitiv måte å skrive det på? F.eks `var date = new DateTime(adjustedFromDate.Year, adjustedFromDate.Month, 1)`?